crypto/keys/types.go

```diff
@@ -46,9 +46,11 @@ type Keybase interface {
 	// The following operations will *only* work on locally-stored keys
 	Update(name, oldpass string, getNewpass func() (string, error)) error
 	Import(name string, armor string) (err error)
+	ImportPrivKey(name string, armor string, passphrase string) error
 	ImportPubKey(name string, armor string) (err error)
 	Export(name string) (armor string, err error)
 	ExportPubKey(name string) (armor string, err error)
+	ExportPrivKey(name string, decryptPassphrase string, encryptPassphrase string) (string, error)
```

Comment:
I am still unsure what is the team preference. Actually, in these lines we see both styles
When several argument have the same time...:
1- group arguments and single type at the end
2- type for each argument?

Reply:
I think the inconsitency here is my fault. I'll fix it and make it consistent with the rest of the interface methods

Reply:
oh, I actually meant this:
```
ExportPrivKey(name string, decryptPassphrase string, encryptPassphrase string) (string, error)
```
vs
```
ExportPrivKey(name, decryptPassphrase, encryptPassphrase string) (string, error)
```
that is something that I was told about myself in other reviews

Reply:
100% the latter. Args with the same type, the type should be omitted until the end.